pkg/models/stats.go

```diff
@@ -18,6 +18,7 @@ type SystemStats struct {
 	Folders                   int64
 	ProvisionedDashboards     int64
 	AuthTokens                int64
+	ApiKeys                   int64
```

Comment:
Nit-pick/for-future-reference: We're not wholly consistent on this but we've been trying to be consistent about upper case initialisms in the vein of the Go project's guidelines. https://github.com/golang/go/wiki/CodeReviewComments#initialisms

```suggestion
	APIKeys                   int64 `xorm:"api_keys"`
```

You don't have to fix this before merging, but if you'd like to you're welcome to do so 🙂

Reply:
I'll change it, I prefer it as `APIKeys` but I saw the model was not declared as that :cry: